elements/tcpudp/shifter.cc

```diff
@@ -66,18 +66,19 @@ inline void Shifter::process(Packet *p) {
         q->rewrite_ipport(htonl(ip_src), htons(sport), 0, ip_proto == IP_PROTO_TCP);
         q->rewrite_ipport(htonl(ip_dst), htons(dport), 1, ip_proto == IP_PROTO_TCP);
     }
+    return q;
```

Comment:
@MassimoGirondi there were a few problems here. uniqueify may return a new packet, so you have to return it and use that one, you can't use the old one afterwards